RPLCD/i2c.py

```diff
@@ -61,18 +109,26 @@ def __init__(self, address, port=1,
         :type auto_linebreaks: bool
         :param backlight_enabled: Whether the backlight is enabled initially. Default: ``True``.
         :type backlight_enabled: bool
-
+        :param i2c_expander: Set your i2c chip type. "PCF8574" or "MCP23008" supported. Default: "PCF8574"
         """
+
         # Set own address and port.
         self.address = address
         self.port = port
 
+        # Set i2c expander, "PCF8574" and "MCP23008" are supported.
+        self._i2c_expander = i2c_expander
```

Comment:
Maybe you could move the expander check from line 130 to here? Makes more sense to do it when assigning the variable, instead of waiting for backlight control to check whether the value is valid.

Something like this:

```python
if i2c_expander in ['PCF8574', 'MCP23008']:
    self._i2c_expander  = i2c_expander
else:
    raise NotImplementedError('I2C expander "%s" is not supported.' % i2c_expander)
```